speclang_modern.tex

```diff
@@ -3389,6 +3387,27 @@ \section{Ghost variables and statements}
   \listinginput{1}{ghostcfg.c}
 \end{example}
 
+The qualifier \lstinline|\ghost| can be used, in ghost code only, in declaration
+of a variable to state precisely in which memory (normal or ghost) its type
+belongs.
+
+As \lstinline|\ghost| is a qualifier, it obeys to the same typing rules except
```

Comment:
"As"  -> "Since" (causal conjunction)

Reply:
Something seems missing inside `ghost.tex` about this `\ghost` qualifier. 

`qualifier ::= C-qualifier`

`| "\ghost" ; qualifier usable only in ghost code`